packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -16,10 +16,39 @@ class _BaseConfig:
     keep_untouched = (cached_property,)
 
 
+class ContainerSpec(BaseModel):
+    """Implements entries can can be overriden for https://docs.docker.com/engine/api/v1.41/#operation/ServiceCreate
+    request body: TaskTemplate -> ContainerSpec
+    """
+
+    command: list[str] = Field(
+        alias="Command",
+        description="The command to be run in the image. If None, it will not override default 'run'",
```

Comment:
Q: should we not have actually "run" as the default?

Reply:
I prefer to do that when I get to the last model's interface i.e. ``Image``. 
At this point, it is part of the ``models_library`` and it is used only as a mean to override.